docs/concepts/pydantic_prompts.md

```diff
@@ -77,14 +77,16 @@ class BookRecommendationPrompt(Prompt):
     topics: list[str]
 	genres: list[str]
 
-	@property
-	def topics_x_genres(self) -> str:
-		"""Returns `topics` as a comma separated list."""
-		return "\n".join([
-				f"Topic: {topic}, Genre: {genre}"
-				for topic in topics
-				for genre in genre
-		])
+    @property
+    def topics_x_genres(self) -> str:
+        """Returns `topics` as a comma separated list."""
+        return "\n\t".expandtabs(4).join(
```

Comment:
only the first item in the list was tabbed, looking like
```
Can you recommend some books on the following topic and genre pairs?
	Topic: coding, Genre: fiction
Topic: coding, Genre: fantasy
Topic: music, Genre: fiction
Topic: music, Genre fantasy
```
so i added a \t and expand tabs